### PR TITLE
Linters file

### PR DESCRIPTION
**Pull request by Boku52500**
Added linters to the project.

### CODE REVIEW

File: .hintrc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "connector": {
+    "name": "local",
+    "options": {
```
Comment (bobb-Rob) [inline .hintrc:4]: - [ ] First of all, I noticed you merged your pull request(PR) without any approval. You should not merge your PR if it has not been approved. Kindly take note of this for subsequent projects.

File: .hintrc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      "pattern": ["**", "!.git/**", "!node_modules/**"]
+    }
+  },
```
Comment (bobb-Rob) [inline .hintrc:7]: - [ ] [REQUIRED] This pull request(PR) can be improved by updating the title and description.
Pull Request should have a descriptive title telling what changes are inside.
Pull Request should have a summary describing introduced changes in more detail.
e.g Title -> Hello Microverse project and the summary -> Mention the things you did in brief detail.

See the image below for an example of a descriptive PR summary 👇

File: index.html [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<h1>Hello,Microverse!</h1>
```
Comment (bobb-Rob) [inline index.html:1]: - [ ] Please add a full Html code as shown below: 

```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>HTML 5 Boilerplate</title>
    <link rel="stylesheet" href="style.css">
  </head>
  <body>
	<h1>Hello,Microverse!</h1>
  </body>
</html>
```

**Review by bobb-Rob — COMMENTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @Boku52500 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Boku52500** [comment]
@bobb-Rob I think I have done all the requests, but with a new repository. Please check out this link. https://github.com/Boku52500/Hello-Microverse/pull/1

Thank you!

**Pull request merged**